templates/ClientOptionsFile.stencil

```diff
@@ -15,22 +15,27 @@ public struct {{ model.name }}Options: AzureClientOptions {
     public let telemetryOptions: TelemetryOptions
     /// Global transport options
     public let transportOptions: TransportOptions
+    /// The default dispatch queue on which to call all completion handler. Defaults to `DispatchQueue.main`.
+    public let dispatchQueue: DispatchQueue?
```

Comment:
Under what situation where we should use a new queue not the `DispatchQueue.main`?

Reply:
Your test app for one :)  Anytime you want to call a bunch of things in the background. The previous behavior of calling all completionHandlers on main would break the chain and cause the code to hang.